### PR TITLE
Ignore InternationalizationTest.userAttributeTest

### LINKED ISSUE #20307: Test `InternationalizationTest` fails in CI
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

testsuite

### Describe the bug

When the test `InternationalizationTest` is executed with Chrome in our CI, it fails.

https://github.com/keycloak/keycloak/actions/runs/4939759509/jobs/8843057671?pr=20293
https://github.com/keycloak/keycloak/actions/runs/4945013478/jobs/8841461993?pr=20301

```
Error: OR] Failures: 
Error: OR] org.keycloak.testsuite.ui.account2.InternationalizationTest.userAttributeTest
Error: OR]   Run 1: InternationalizationTest.userAttributeTest:105 expected:<[English]> but was:<[und]>
Error: OR]   Run 2: InternationalizationTest.userAttributeTest:105 expected:<[English]> but was:<[und]>
Error: OR]   Run 3: InternationalizationTest.userAttributeTest:105 expected:<[English]> but was:<[und]>
```

Stack trace:
```
--------- org.keycloak.testsuite.ui.account2.InternationalizationTest output end  ---------
Error:  Tests run: 6, Failures: 3, Errors: 0, Skipped: 2, Time elapsed: 43.433 s <<< FAILURE! - in org.keycloak.testsuite.ui.account2.InternationalizationTest
Error: OR] org.keycloak.testsuite.ui.account2.InternationalizationTest.userAttributeTest  Time elapsed: 8.489 s  <<< FAILURE!
-- org.junit.ComparisonFailure: expected:<[English]> but was:<[und]>
-- 	at org.junit.Assert.assertEquals(Assert.java:117)
-- 	at org.junit.Assert.assertEquals(Assert.java:146)
-- 	at org.keycloak.testsuite.ui.account2.InternationalizationTest.userAttributeTest(InternationalizationTest.java:105)
-- 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
-- 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
-- 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
-- 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
-- 	at org.jboss.arquillian.junit.Arquillian$8$1.invokeMethod(Arquillian.java:325)
-- 	at org.jboss.arquillian.junit.MethodInvoker$1.invoke(MethodInvoker.java:18)
-- 	at org.jboss.arquillian.container.test.impl.execution.LocalTestExecuter.execute(LocalTestExecuter.java:57)
-- 	at org.keycloak.testsuite.arquillian.ModelTestExecutor.execute(ModelTestExecutor.java:51)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor173.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:133)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:105)
-- 	at org.jboss.arquillian.core.impl.EventImpl.fire(EventImpl.java:62)
-- 	at org.jboss.arquillian.container.test.impl.execution.ClientTestExecuter.execute(ClientTestExecuter.java:50)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor172.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:[568](https://github.com/keycloak/keycloak/actions/runs/4939759509/jobs/8843057671?pr=20293#step:4:569))
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
-- 	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createContext(ContainerEventController.java:128)
-- 	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createTestContext(ContainerEventController.java:118)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor171.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createTestContext(TestContextHandler.java:116)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor7.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(TestContextHandler.java:83)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(TestContextHandler.java:69)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:133)
-- 	at org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.test(EventTestRunnerAdaptor.java:139)
-- 	at org.jboss.arquillian.junit.MethodInvoker.invoke(MethodInvoker.java:15)
-- 	at org.jboss.arquillian.junit.Arquillian$8.evaluate(Arquillian.java:332)
-- 	at org.jboss.arquillian.junit.Arquillian$4.evaluate(Arquillian.java:204)
-- 	at org.jboss.arquillian.junit.Arquillian.multiExecute(Arquillian.java:350)
-- 	at org.jboss.arquillian.junit.Arquillian.access$200(Arquillian.java:54)
-- 	at org.jboss.arquillian.junit.Arquillian$5.evaluate(Arquillian.java:215)
-- 	at org.jboss.arquillian.junit.Arquillian$7$1.invoke(Arquillian.java:279)
```

### Version

999

### Expected behavior

Test is ok

### Actual behavior

Test is NOT ok

### How to Reproduce?

```
mvn -f testsuite/integration-arquillian/tests/other/base-ui/pom.xml test -Dtest=InternationalizationTest -Dbrowser=chrome
```

### Anything else?

_No response_

### LINKED ISSUE #20349: WebAuthn test fails in the GHA
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

testsuite

### Describe the bug

https://github.com/keycloak/keycloak/actions/runs/4978190160/jobs/8908431577?pr=20347
https://github.com/keycloak/keycloak/actions/runs/4956553186/jobs/8867413465?pr=20325
https://github.com/keycloak/keycloak/actions/runs/4955990654/jobs/8866088248?pr=20323

```
Error: OR] Failures: 
Error: OR] org.keycloak.testsuite.webauthn.account.WebAuthnSigningInTest.checkAuthenticatorTimeLocale
Error: OR]   Run 1: WebAuthnSigningInTest.checkAuthenticatorTimeLocale:336 
-- Expected: is "May 15, 2023, 8:52 AM"
--      but: was "2023 May 15 08:52"
Error: OR]   Run 2: WebAuthnSigningInTest.checkAuthenticatorTimeLocale:336 
-- Expected: is "May 15, 2023, 8:55 AM"
--      but: was "2023 May 15 08:55"
Error: OR]   Run 3: WebAuthnSigningInTest.checkAuthenticatorTimeLocale:336 
-- Expected: is "May 15, 2023, 8:56 AM"
--      but: was "2023 May 15 08:56"
-- [INFO] 
-- [INFO] 
Error: OR] Tests run: 124, Failures: 1, Errors: 0, Skipped: 15
```

### Version

999

### Expected behavior

Test works

### Actual behavior

Test does NOT work

### How to Reproduce?

Execute the `WebAuthnSigningInTest#checkAuthenticatorTimeLocale` test

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by miquelsi**
Closes #20307 and #20349

### CODE REVIEW

**Comment by vmuzikar** [comment]
@miquelsi Could you please create a follow-up issue with 22.0.0 milestone to re-enable the test?

**Comment by miquelsi** [comment]
Follow up issue targeting 22.0 #20348

**Review by vmuzikar — DISMISSED** [review]
Looks good from my perspective, thanks @miquelsi!

I think we also need an approval from @keycloak/ui-maintainers.

**Comment by vmuzikar** [comment]
@miquelsi Should we maybe ignore some WebAuthn tests too to fix https://github.com/keycloak/keycloak/issues/20349 in this PR?

**Review by vmuzikar — DISMISSED** [review]
LGTM (provided tests will pass).

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java [hunk 1]
```diff
@@ -260,6 +260,7 @@ public void availableAuthenticatorsAfterRemove(){
     }
 
     @Test
+    @Ignore // TODO: Enable once chromedriver version 113.0.5672.92 is available in https://chromedriver.storage.googleapis.com/
```
Comment (mabartos) [inline testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java:263]: @miquelsi AFAIK, the import for the `@Ignore` annot is not present.

`import org.junit.Ignore;`

Comment (miquelsi) [reply testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java:264]: You're right, my IDE confused me :-)

**Review by mabartos — APPROVED** [review]
@miquelsi Thanks!

**Review by vmuzikar — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**